tools/clang/test/SemaHLSL/hlsl/objects/texture/CalculateLODWithSamplerComparisonState.hlsl

```diff
@@ -35,12 +35,3 @@ float ps(float a:A) : SV_Target {
     t.CalculateLevelOfDetailUnclamped(s, a); // expected-error {{overload of intrinsic CalculateLevelOfDetailUnclamped with SamplerComparisonState requires shader model 6.8 or greater}}
 }
 
-// expected-note@+3{{declared here}}
-// expected-note@+2{{declared here}}
-[shader("vertex")]
-float4 vs(float a:A) : SV_Position {
-  // expected-error@+1 {{Intrinsic CalculateLevelOfDetail potentially used by vs requires derivatives - only available in pixel, compute, amplification, mesh, or broadcast node shaders}}
-  return t.CalculateLevelOfDetail(s, a) + // expected-error {{overload of intrinsic CalculateLevelOfDetail with SamplerComparisonState requires shader model 6.8 or greater}}
-  // expected-error@+1 {{Intrinsic CalculateLevelOfDetailUnclamped potentially used by vs requires derivatives - only available in pixel, compute, amplification, mesh, or broadcast node shaders}}
-    t.CalculateLevelOfDetailUnclamped(s, a); // expected-error {{overload of intrinsic CalculateLevelOfDetailUnclamped with SamplerComparisonState requires shader model 6.8 or greater}}
-}
```

Comment:
The bug we are fixing is that this code produced defaultError warnings before on previous shader models. We don't have a test here that verifies that they don't anymore. Something like this, but using a non-comparison sampler that just verifies there are no diagnostics should be sufficient.

Reply:
Added.

Reply:
I love the new test that was added, but I didn't mean to imply that this code should be removed. It's perfectly fine, I just think that we needed another test.